File: server/router/admin/commerce_cdk.go
package admin

import (
	"github.com/gin-gonic/gin"
	"server/controller/admin/commerce_cdk"
	"server/global"
	"server/router/middleware"
)

func InitCommerceCdk(group *gin.RouterGroup) {
	g := group.Group("commerce/cdk", middleware.Auth(global.Jwt), middleware.AuthAdmin())
	g.POST("create", commerce_cdk.Create)
	g.POST("page", commerce_cdk.Page)
	g.POST("disable", commerce_cdk.Disable)
	g.POST("batch-disable", commerce_cdk.BatchDisable)
	g.POST("batch-enable", commerce_cdk.BatchEnable)
	g.POST("batch-delete", commerce_cdk.BatchDelete)
	g.POST("update-remark", commerce_cdk.UpdateRemark)
}

File: web/src/api/admin/commerce_cdk.js
import request from "../../request/index.js";
const baseUrl = '/v1/admin/commerce/cdk'
export const apiAdminCommerceCdkCreate = (data) => request.request({url: `${baseUrl}/create`, method: 'POST', data})
export const apiAdminCommerceCdkPage = (data) => request.request({url: `${baseUrl}/page`, method: 'POST', data})
export const apiAdminCommerceCdkDisable = (data) => request.request({url: `${baseUrl}/disable`, method: 'POST', data})
export const apiAdminCommerceCdkBatchDisable = (data) => request.request({url: `${baseUrl}/batch-disable`, method: 'POST', data})
export const apiAdminCommerceCdkBatchEnable = (data) => request.request({url: `${baseUrl}/batch-enable`, method: 'POST', data})
export const apiAdminCommerceCdkBatchDelete = (data) => request.request({url: `${baseUrl}/batch-delete`, method: 'POST', data})
export const apiAdminCommerceCdkUpdateRemark = (data) => request.request({url: `${baseUrl}/update-remark`, method: 'POST', data})
